### PR TITLE
BL-1030: Take care of some typescript problems webstorm was complaining ...

### PR DESCRIPTION
**Pull request by phillip-hopper**
...about

### CODE REVIEW

**Comment by sillsdevgerrit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.lsdev.sil.org//job/GitHub-Bloom-Wrapper-master-debug/315/
Test PASSed.

File: src/BloomBrowserUI/lib/localizationManager/localizationManager.js [hunk 1]
```diff
@@ -179,39 +180,30 @@ var LocalizationManager = (function () {
...
+    // *      });
+    //
+    //*/
+    //asyncGetText(id: string, englishText: string, langId: string, ...args): JQueryPromise {
+    //    var deferred = new $.Deferred();
+    //    var promise = this.asyncGetTextInLang(id, englishText, "UI", args);
+    //    promise.done(text => deferred.resolve(text));
+    //    promise.fail(this.asyncGetTextInLang(id, englishText, "en", args).done(text => deferred.resolve(text)));
+    //    return deferred.promise();
+    //}
```
Comment (JohnThomson) [inline src/BloomBrowserUI/lib/localizationManager/localizationManager.js]: Is there a reason to keep this commented-out code?

Comment (phillip-hopper) [inline src/BloomBrowserUI/lib/localizationManager/localizationManager.js]: Not really. Removed.

**Comment by sillsdevgerrit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.lsdev.sil.org//job/GitHub-Bloom-Wrapper-master-debug/327/
Test PASSed.

**Pull request merged**